### PR TITLE
initial version of supplier api sdk

### PR DESCRIPTION
**Pull request by adombrovsky**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,41 +1,2 @@
...
-/app/phpunit.xml
-/phpunit.xml
-
-# Build data
-/build/
-
-# Backup entities generated with doctrine:generate:entities command
-**/Entity/*~
+vendor
+git_push.sh
```
Comment (aazon) [inline .gitignore]: Please remove `git_push.sh` and add `composer.lock`

Comment (adombrovsky) [inline .gitignore]: add composer.lock to gitignore ?

Comment (adombrovsky) [inline .gitignore]: @aazon added to gitignore and removed from repo

File: autoload.php [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<?php
```
Comment (adombrovsky) [inline autoload.php]: removed

File: docs/Api/InventoryApi.md [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+// Configure API key authorization: apiKey
+SMS\Suppliers\Configuration::getDefaultConfiguration()->setApiKey('api-key', 'YOUR_API_KEY');
+// Uncomment below to setup prefix (e.g. Bearer) for API key, if needed
+// SMS\Suppliers\Configuration::getDefaultConfiguration()->setApiKeyPrefix('api-key', 'Bearer');
+// Configure API key authorization: apiUsername
+SMS\Suppliers\Configuration::getDefaultConfiguration()->setApiKey('api-username', 'YOUR_API_KEY');
+// Uncomment below to setup prefix (e.g. Bearer) for API key, if needed
+// SMS\Suppliers\Configuration::getDefaultConfiguration()->setApiKeyPrefix('api-username', 'Bearer');
+
+$api_instance = new SMS\Suppliers\Api\InventoryApi();
```
Comment (adombrovsky) [inline docs/Api/InventoryApi.md]: I can't fix it now, because it's autogenerated, but I'll find a solution later.

**Pull request merged**